libs/SalesforceSDKCore/SalesforceSDKCore/Classes/NativeLoginManager.swift

```diff
@@ -71,4 +71,36 @@ public protocol NativeLoginManager {
     ///
     /// Note: this call will dismiss your login view controller.
     @objc func biometricAuthenticationSuccess()
+    
+    // MARK: Headerless, Password-Less Login Via One-Time-Passcode
+    
+    /// Submits a request for a one-time-passcode to the Salesforce headerless password-less login flow.
```

Comment:
"headerless" should be "headless" I assume.

Reply:
I have proven unable to type headless this past week. Thanks!